beta/src/pages/learn/your-first-component.md

```diff
@@ -147,19 +142,22 @@ export default function Gallery() {
 ```
 
 ```css
-img { margin: 0 10px 10px 0; height: 90px; }
+img {
```

Comment:
same as above. lets not change any code alignments

```
img { margin: 0 10px 10px 0; height: 90px; }
```

Reply:
This is not fixed correctly. You can copy the original version from left side [here](https://github.com/reactjs/hi.reactjs.org/pull/204/files#diff-987109509fdb76c4b375a50d1545ac66a321a4dbfb786d5fe75c84760e30b7fdL150)